### PR TITLE
bbrooks/4361-Fix-Packer-Validation-AMI-Builds

### PR DESCRIPTION
**Pull request by cthulhuplus**
Resolves # 4361

### Description
The Packer Validation & Packer AMI workfllows are broken. Update them so that they are functional again.

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |

### Steps to manually verify this change

1. Packer Validation passes
2. Packer AMI builds work
3.

### This pull request is ready to code review when

- [ ] Automated tests are updated (and all tests are passing)
- [ ] New automated test cases are documented above
- [ ] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [ ] Associated OpenAPI documentation has been updated
- [ ] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [ ] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by cthulhuplus** [comment]
```
packer validate -var-file="/mnt/c/Users/brook/Git/hashi_files/packer/app-preview.hcl" preview-ami.pkr.hcl
The configuration is valid.

packer validate -var-file="/mnt/c/Users/brook/Git/hashi_files/packer/backend.hcl
" backend-ami.pkr.hcl
The configuration is valid.

packer validate -var-file="/mnt/c/Users/brook/Git/hashi_files/packer/mongo-prod.
hcl" mongo-ami.pkr.hcl
The configuration is valid.
```

**Comment by cthulhuplus** [comment]
```
==> Builds finished. The artifacts of successful builds are:
--> amazon-ebs.Golden_Image: AMIs were created:
us-east-1: ami-0970fddb6990a992b
```

File: packer/preview-ami.pkr.hcl [hunk 1]
```diff
@@ -57,9 +57,7 @@ build {
             "OKTA_API_KEY=${var.preview_okta_api_key}",
             "ENVIRONMENT=${var.environment}",
             "PREVIEW_TEALIUM_TAG=${var.preview_tealium_tag}",
-            "PREVIEW_TEALIUM_ENV=${var.preview_tealium_env}",
-            "PREVIEW_LD_API_KEY=${var.preview_ld_api_key}",
-            "PREVIEW_LD_CLIENT_ID=${var.preview_ld_client_id}"
+            "PREVIEW_TEALIUM_ENV=${var.preview_tealium_env}"
```
Comment (tbolt) [inline packer/preview-ami.pkr.hcl:60]: why remove the launch darkly keys?

**Review by tbolt — APPROVED** [review]

File: packer/preview-ami.pkr.hcl (hunk 1 above)
Comment (cthulhuplus) [reply packer/preview-ami.pkr.hcl:60]: They aren't used by the AMI creation process. I could add them, but they would still not get used. I chose to remove them to more accurately what is needed and is used by the AMI creation process.

Comment (tbolt) [reply packer/preview-ami.pkr.hcl:60]: that makes sense, I like that

**Comment by mirano-darren** [comment]
Looks good to me! All packer files validated and preview built successfully.

**Review by mirano-darren — APPROVED** [review]

**Pull request merged**